src/pal/src/init/pal.cpp

```diff
@@ -742,32 +741,17 @@ PAL_EntryPoint(
 
 /*++
 Function:
-  PALCommonCleanup
-
-Utility function to prepare for shutdown.
+  PAL_Shutdown
 
+Abstract:
+  This function is the called. It shuts down the PAL WITHOUT exiting the current process.
 --*/
-void 
-PALCommonCleanup()
+void
+PALAPI
+PAL_Shutdown(
```

Comment:
I put it in at first so I could send the DLL_PROCESS_DETACH notifications on shutdown but that has too many problems.  I left it in for symmetry with PAL_Initialize and for any future cleanup, etch.  It currently cleans up exactly what ExitProcess/TerminateProcess does.

mikem